packages/fast-components-react-base/src/auto-suggest/auto-suggest.props.ts

```diff
@@ -6,6 +6,11 @@ import {
 import { ListboxItemProps } from "../listbox-item";
 import { AutoSuggestState } from "./auto-suggest";
 
+export enum MenuTriggers {
```

Comment:
Shouldn't the choice to open the menu on focus be in addition to click and keystrokes?  Maybe this should just be a Boolean?

Reply:
What about other scenarios? Like character match? This gives us more flexibility to add on functionality in the future.

Reply:
What's character match?

Reply:
I guess "should the menu open on focus" seems like yes/no option to me.

Reply:
Character match is another potential "MenuTrigger" where the menu will only open when a certain number of inputed characters match. this is all from the aria spec linked in the issue this PR is resolving.
https://github.com/microsoft/fast-dna/issues/2458

Reply:
Chatted with @scomea offline and decided its best for now to remove `MenuTriggers` from this PR until we discuss in more detail. The filtering / storing the search string was the most important bits of this PR anyway.